src/components/Button.js

```diff
@@ -173,14 +173,11 @@ class Button extends Component {
             return <ContentComponent />;
         }
 
-        if (this.props.isLoading) {
-            return <ActivityIndicator color={themeColors.textReversed} />;
-        }
-
         const textComponent = (
             <Text
                 selectable={false}
                 style={[
+                    this.props.isLoading && styles.opacity0,
```

Comment:
using opacity 0 as visibility hidden doesn't work on native

Reply:
I can't understand the question here. Can you rephrase it?

Reply:
> using opacity 0 as visibility hidden doesn't work on native

What have you done instead here to solve the problem?

Reply:
I think you misunderstood the comment, let me rephrase it for you. "Visibility hidden doesn't work on native so instead of it I am using opacity 0"

Reply:
There's a lot rephrases here 😄